app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+class Animal:
+    animals = []
+
+    def __init__(self, name: str,
+                 appetite: int = 0,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.animals.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name)
+        self.is_hungry = is_hungry
+        self.appetite = 3
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name)
+        self.is_hungry = is_hungry
+        self.appetite = 7
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
+    hungry = 0
```

Comment:
You can try to do this logic in 1 line using `sum()` method and `comprehension `